### ISSUE TITLE
Space characters not displayed

### ISSUE THREAD

**Issue by AlbertoTrigo** [LordRusk/gocaudices#3]
By default, the code is trimming space characters in the blocks. I've seen the function  ```barBytesArr[b.pos] = bytes.TrimSpace(outputBytes)``` in line 37. However I don't know how to disable this "feature", and would like to separate the blocks using space characters.

Thanks for the software.

**Comment by LordRusk** [comment]
I'm sorry I wasn't able to get to this sooner, but I hope I can still help.

The reason I trim space is in case there are any newline `\n characters from the commands output, if this happens, at least with versions of dwm I forked and used at the time, it will fuck the bar.

The `delim` var in `blocks.go` is where you are supposed to set the delimiter, since the delimiter is added when the bar is turned into a []byte.

With this in mind I'm not quite sure what you mean by using space characters to separate blocks, that is actually the default delimiter.

Keep in mind I made this to be as simple and hackable as possible, so if you want to "disable this feature"🙄 then remove the `bytes.TrimSpace`.
